setup-disk.in

```diff
@@ -459,6 +462,7 @@ install_mounted_root() {
 	case "$BOOTLOADER" in
 		grub) setup_grub "$mnt" "$root" "$modules" "$kernel_opts" "$bootdev" $disks ;;
 		syslinux) setup_syslinux "$mnt" "$root" "$modules" "$kernel_opts" "$bootdev" ;;
+		zipl) break;;
```

Comment:
What loop are you trying to break here? o.O

Reply:
Break from the case-esac :D Anyway that was updated in new commit.